openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx

```diff
@@ -616,6 +643,7 @@ const DatabaseSchemaPage: FunctionComponent = () => {
         activeTabHandler(1);
       }
       getDetailsByFQN();
+      getSchemaTables(INITIAL_PAGING_VALUE);
```

Comment:
I would recommend only making this api call if databaseSchema data is available.

Reply:
Made the suggested changes. Can you please review?